### PR TITLE
adds automatic cloning pod for pod wars.

### PR DESCRIPTION
**Pull request by ThePotato97**
## About the PR 

Adds an automatic cloner

## Why's this needed? 

Cause @Kyle2143 wanted it

### CODE REVIEW

File: code/obj/machinery/clonepod.dm [hunk 1]
```diff
@@ -1152,3 +1152,28 @@
...
+
+/obj/machinery/clonepod/automatic/disposing()
+	..()
+	UnsubscribeProcess()
+
+/obj/machinery/clonepod/automatic/proc/spawn_a_ghost()
+	for(var/mob/dead/observer/ghost in mobs)
+		var/datum/mind/ghost_mind = ghost.mind
+		if(ghost.client && !ghost_mind.dnr)
+			spawn(growclone(ghost, ghost.real_name, ghost_mind))
```
Comment (Kyle2143) [inline code/obj/machinery/clonepod.dm:1178]: Not sure why the spawn is written this way. Idk if a spawn is necessary though, and i don't think it's exacuting it in paralell like spawns do.

File: code/obj/machinery/clonepod.dm [hunk 2]
```diff
@@ -1152,3 +1152,28 @@
...
+	..()
+	animate_rainbow_glow(src) // rgb shit cause it looks cool
+	SubscribeToProcess()
+
+/obj/machinery/clonepod/automatic/disposing()
+	..()
+	UnsubscribeProcess()
+
+/obj/machinery/clonepod/automatic/proc/spawn_a_ghost()
+	for(var/mob/dead/observer/ghost in mobs)
```
Comment (Kyle2143) [inline code/obj/machinery/clonepod.dm:1175]: I think, unless something has changed, process ticks every second or two. This is a lot ofloops through world in that case.

**Review by Kyle2143 — CHANGES_REQUESTED** [review]
Good stuff. Left two comments about review that I thought could be improved though. If you want to look at em, go ahead, if not I'll make changes after merge if they do turn out to be issues that need changing.

File: code/obj/machinery/clonepod.dm (hunk 2 above)
Comment (ThePotato97) [reply code/obj/machinery/clonepod.dm:1175]: Could maybe make it check every 10 seconds.

**Pull request merged**